src/models/room.ts

```diff
@@ -914,12 +914,38 @@ export class Room extends ReadReceipt<RoomEmittedEvents, RoomEventHandlerMap> {
         return this.myUserId;
     }
 
+    // gets service members (e.g. helper bots) for exclusion
```

Comment:
For exclusion from what?

Might as well make this a proper doc-comment

```suggestion
    /**
     * Gets the "functional members" in this room.
     *
     * Returns the list of userIDs from the `io.element.functional_members` event. Does not consider the
     * current membership states of those users.
     *
     * @see https://github.com/element-hq/element-meta/blob/develop/spec/functional_members.md.
     */
```